nodebuilder/tests/tastora/e2e_sanity_test.go

```diff
@@ -0,0 +1,257 @@
+//go:build integration
```

Comment:
Should we change the tag to be aligned with how we call the tests? sanity or smoke

Reply:
currently we use integration as default tag throughout. i would avoid having to maintain multiple tags. we can come back to this when we have different tastora tests and can clearly separate them into smoke vs api vs integration tests.